src/web/settingsview.mjs

```diff
@@ -13,13 +14,19 @@ class SettingsView {
    */
   #aniTime = 250;
 
+  /**
+   * @private {string} savedSettingsLink - set time to show/hide elements
```

Comment:
Please update the explanation of the savedSettingsLink. It now looks like a copy/paste error from the docs of aniTime